regression/cbmc-java/invalid_classpath/test-jar.desc

```diff
@@ -0,0 +1,10 @@
+CORE symex-driven-lazy-loading-expected-failure
+Test.class
+--classpath ./NotHere.jar
+^EXIT=6$
+^SIGNAL=0$
+^the program has no entry point$
+^failed to load class `Test'$
+--
```

Comment:
This is a bit cryptic, so worth explaining what you're testing for and why this is the expected behaviour (perhaps in a followup PR considering the urgency)